CRM/Report/Form/Campaign/SurveyDetails.php

```diff
@@ -204,8 +209,18 @@ public function __construct() {
             'operatorType' => CRM_Report_Form::OP_MULTISELECT,
             'options' => $resultOptions,
           ),
+          'activity_date_time' => array(
+            'title' => ts('Date'),
+            'type' => CRM_Utils_Type::T_DATE,
```

Comment:
Unfortunately, this doesn't work correctly.  If you look at the developer tab, you can see they're searching by date (`20170821` to `20170821`) and not datetime (`20170821000000` to `20170821235959`).